.github/workflows/docker-deploy.yml

```
@@ -21,20 +21,31 @@ jobs:
+        run: npm ci
+      
+      - name: Build application
+        run: npm run build
       
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
         with:
           install: true
       
+      - name: Cache Docker layers
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)